src/rebuild/srv.c

```diff
@@ -461,31 +461,31 @@ ds_rebuild_query(uuid_t pool_uuid, struct daos_rebuild_status *status)
 	/* If there are still rebuild task queued/running for the pool, let's reset
 	 * the done status.
 	 */
-	if (status->rs_done == 1 &&
+	if (status->rs_state == DRS_COMPLETED &&
 	    (!d_list_empty(&rebuild_gst.rg_queue_list) ||
 	     !d_list_empty(&rebuild_gst.rg_running_list))) {
 		struct rebuild_task *task;
 
 		d_list_for_each_entry(task, &rebuild_gst.rg_queue_list, dst_list) {
 			if (uuid_compare(task->dst_pool_uuid, pool_uuid) == 0) {
-				status->rs_done = 0;
+				status->rs_state = DRS_IN_PROGRESS;
```

Comment:
now that we have two fields, do we need to set both so that old clients still work?